src/Tickets/Blocks/Tickets/app/editor/inactive/inactive.js

```diff
@@ -20,7 +20,7 @@ const InactiveTickets = ( { title } ) => {
 			className="tribe-editor__card-no-bottom-border"
 			header={ __( 'Tickets', 'event-tickets' ) }
 		>
-			<div className="tickets-description tickets-row-line">{ title }</div>
+			<div className="tickets-description tickets-row-line" dangerouslySetInnerHTML={{__html: title }} />
```

Comment:
Avoid `dangerouslySetInnerHTML`

Reply:
dangerouslySetInnerHTML was avoided. Now a ReactNode is being used to display text and the translation was split